airflow/www/templates/airflow/dag.html

```diff
@@ -322,6 +327,7 @@ <h4 class="modal-title" id="dagModalLabel">
 {% endblock %}
 {% block tail %}
   {{ super() }}
+  <script src="{{ url_for_asset('utils.js') }}"></script>
```

Comment:
We're including this file in tail_js - do we need to include it again?

Reply:
We have already discussed this in the past.
https://github.com/apache/airflow/pull/4787#discussion_r267573261